osu.Game.Rulesets.Osu/Edit/OsuSelectionHandler.cs

```diff
@@ -207,28 +191,119 @@ public override bool HandleRotation(float delta)
             return true;
         }
 
-        private bool moveSelection(Vector2 delta)
+        private void scaleSlider(Slider slider, Vector2 scale)
```

Comment:
Was there any reason not to use `getClampedScale()` here? Because it isn't used, sliders can still be "flipped" or resized past the playfield size.

That's about the only remaining issue I have managed to find after some fiddling, though. Looks solid to me otherwise :+1:

Reply:
When you say "resize past the playfield size", you are talking about the slider body, correct? I wasn't really concerned with keeping the body in the playfield since it isn't respected by movement either in stable or lazer. I figured that was the intended behavior.

`getClampedScale()` won't work directly in this case, since the method of scaling is to different from multi-object scaling. The original plan was to create some kind of overload for use in `scaleSlider()` before I figured out that limiting the max scale (as in finding the closest possible scale where sliderhead and sliderend stay in bounds) isn't feasible here.

I limited the minimum size for single slider scaling.